assets/css/custom.css

```diff
@@ -530,34 +530,79 @@ section.speakers .qcTabTitle .section-title {
 
 /* VENUE SECTION STYLES */
 
-.venue-section {
+.location-section {
 	background-color: #F1D8D3;
 }
 
-.venue-section .venue-info-bg {
+.location-section .venue-info-bg {
 	background-color: #172751;
 }
 
-.venue-section .venue-info-map {
+.location-section .venue-info-map {
 	margin: 0;
 }
 
-.venue-section .venue-info-bg {
+.location-section .venue-info-bg {
 	padding: 3em;
 	color: #F1D8D3;
 	line-height: 2.4;
 }
 
-.venue-section iframe {
+.location-section iframe {
 	width: 100%;
 	height: 100%;
 	pointer-events: none;
 }
 
-.venue-section .venue-info-content a:hover {
+.location-section .venue-info-content a:hover {
 	color: #F1D8D3;
 }
 
+.location-section .hotel-section{
+  color: #172751;
+  background: #fff;
+  padding: 20px;
+}
+
+.location-section .hotel-blurb{
+  font-size: 18px;
+  color: #172751;
+  padding-top: 20px;
+}
+.location-section .hotel-info{
+  font-size: 17px;
+}
```

Comment:
Can we make the `line-height` of the hotel info `1.5em`? I think that'll be a little easier to read